packages/pages/src/util/yextrcContents.test.ts

```diff
@@ -0,0 +1,18 @@
+import fs from "fs";
+import { parseYextrcContents } from "./yextrcContents.js";
+
+describe("parseYextrcContents", () => {
+  it("parse valid accountId and universe from .yextrc", () => {
+    const providedAccountId = 1089287;
+    const providedUniverse = "prod";
+    fs.writeFileSync(
+      ".yextrc",
+      `accountId: ${providedAccountId}\nuniverse: ${providedUniverse}`
+    );
+
+    const { accountId, universe } = parseYextrcContents();
```

Comment:
Let's have this return undefined for both instead of throwing. It will make usage of it cleaner.